src/Simplex/Chat/View.hs

```diff
@@ -1210,9 +1228,15 @@ viewChatError logLevel = \case
     CENoConnectionUser agentConnId -> ["error: message user not found, conn id: " <> sShow agentConnId | logLevel <= CLLError]
     CEActiveUserExists -> ["error: active user already exists"]
     CEUserExists name -> ["user with the name " <> ttyContact name <> " already exists"]
+    CEUserUnknown -> ["user does not exist or incorrect password"]
     CEDifferentActiveUser commandUserId activeUserId -> ["error: different active user, command user id: " <> sShow commandUserId <> ", active user id: " <> sShow activeUserId]
     CECantDeleteActiveUser _ -> ["cannot delete active user"]
     CECantDeleteLastUser _ -> ["cannot delete last user"]
+    CECantHideLastUser _ -> ["cannot hide last visible user"]
```

Comment:
just got confused by the text, as if it's last active. ok to keep as is
```suggestion
    CECantHideOnlyUnhiddenUser _ -> ["cannot hide the only unhidden user"]
```

Reply:
updated